bl-service/src/main/java/com/ayeeti/blservice/location/requests/LocationRequest.java

```diff
@@ -0,0 +1,17 @@
+package com.ayeeti.blservice.location.requests;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Builder
+@Getter
+@Setter
+@AllArgsConstructor
+public class LocationRequest {
```

Comment:
Not sure why you used class here. We can use record here as for all other Requests.